.github/workflows/track-shipping-version.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  release:
+    types: [released]
+
+permissions:
+  contents: write
+
+env:
+  shipping_branch_prefix: 'shipping'
+
+jobs:
+  update-shipping-branch:
+    if: github.repository == 'dotnet/dotnet-monitor'
+    name: '[${{ github.ref_name }}] Update shipping branch'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Update shipping branch
+        run: |
+          release_version=${GITHUB_REF_NAME%-*}
+          if [[ ! "$release_version" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+              echo "Unexpected release tag: $release_version."
+              exit 1
+          fi
+
+          major_minor_version=${release_version%.*}
+          shipping_branch_name="${{ env.shipping_branch_prefix }}/$major_minor_version"
+
+          # This is a shallow clone so we will always create a new local branch even if it already exists on the remote
+          git checkout -b "$shipping_branch_name"
+          git push --force --set-upstream origin "HEAD:$shipping_branch_name"
```

Comment:
We'll also have to mirror these branches with a registration here: https://github.com/dotnet/versions/blob/main/Maestro/subscriptions.json#L447